src/target/page/TargetPage.test.tsx

```diff
@@ -1,14 +1,16 @@
 import React from 'react';
 import { mount } from 'enzyme';
 import { act } from 'react-dom/test-utils';
-import api, { Target } from '../../common/utils/api/api';
+import api, { Target, SourceLinkTypes } from '../../common/utils/api/api';
 import TargetPage from './TargetPage';
 
 const testTarget: Target = {
   id: 'tprek:1000',
   name: 'Test target',
   address: 'Main street, Helsinki',
   description: 'Test target description',
+  // eslint-disable-next-line @typescript-eslint/camelcase
+  links: [{ link_type: SourceLinkTypes.ADMIN, url: 'http://local.trek:3001' }],
```

Comment:
Dummie question: why we need link_type in snake_case?

Reply:
I thought that it could be as close as possible with the API response format without any unnecessary abstraction. 
But having said that, I could try some data conversion (api->client) and see how does it look..

Reply:
I fear we end up in client codebase with 2 types of variable naming, which would lead to confusion as to which one to use hmm

Reply:
I think we can go with creating new linter rule or something